### DISCUSSION TITLE
glb file doesn't support animation on Android but on iOS glb file shows animation like blinking, smile, etc.

### DISCUSSION THREAD

**Discussion by Amitdygon** [sceneview/sceneform-android#296] [Q&A] [answered]
Why .glb file does not support animation for AR face but on iOS its worked, animation like blinking , smiling, laughing, etc.

**Comment by grassydragon** [comment, marked as answer]
Hi!
You need to start the animation by yourself. See this sample application: https://github.com/SceneView/sceneform-android/blob/master/samples/ar-model-viewer-java/src/main/java/com/google/ar/sceneform/samples/gltf/MainActivity.java#L136
